api/src/main/java/com/basistech/rosette/api/RosetteAPI.java

```diff
@@ -137,9 +137,11 @@ public RosetteAPI(String key) throws IOException, RosetteAPIException {
      * @throws IOException         General IO exception
      */
     public RosetteAPI(String key, String alternateUrl) throws IOException, RosetteAPIException {
+        boolean shouldCheckVersion = false;
         urlBase = alternateUrl;
         if (urlBase == null) {
-            checkVersionCompatibility();
+            urlBase = DEFAULT_URL_BASE;
```

Comment:
I don't feel like correcting a user error here. If someone wants to use an altUrl, it'd better be a good one and we blow up if not. If UX is a concern here, I'd rather do a URL syntax validation than defaulting to default.